### ISSUE TITLE
Dark Mode is not working correctly on the "pop ups"

### ISSUE THREAD

**Issue by darkfire006** [sdcervi/PokeDexApp#77]
the pop ups like when right clicking/long clicking on a pokemon, and the "how to use" button on the main pokedex page dont have a dark background which means the now white text cant be read

**Comment by sdcervi** [comment]
Thanks for letting me know. What device and browser are you using, so that I can more effectively test the fix?

**Comment by darkfire006** [comment]
I have tried it on win 10-chrome and win 11-edge with the same results

**Comment by sdcervi** [comment]
Thank you, I'll let you know when I have a fix ready.

**Comment by sdcervi** [comment]
Can you take a look at this test URL and let me know if it's fixed?

https://living-pokedex-tracker-5c0b4--darkmode-s0ntpyz4.web.app/pokedex/

**Comment by darkfire006** [comment]
that fixed it for the pop ups like I mentioned. There is one more spot that doesnt look quite right in dark mode, the supporting members list on the donate page:
![image](https://github.com/sdcervi/PokeDexApp/assets/8632995/db7cf87b-81b3-4871-a5bb-edbf3a344af3)

**Comment by sdcervi** [comment]
Good catch! Thanks for spotting that. I think I have it fixed now. Provided everything looks good, I can get the changes published tomorrow.

**Comment by darkfire006** [comment]
Looks good :)
